.github/workflows/test.yml

```diff
@@ -51,11 +51,14 @@ jobs:
         cargo install --version 0.9.1 wasm-pack
     - name: Build
       run: cargo build --verbose $TARGET
+    - name: Disable benchmarking dependencies
+      if: matrix.mode != 'native' || matrix.rust-version == '1.33.0'
```

Comment:
Maybe check for `(matrix.rust-version != 'stable' && matrix.rust-version != 'nightly')` here? That would be one less place to update if the MSRV gets changed. Although I'd also be fine with dropping nightly from benchmarking.